Tasks/AzureAppServiceManageV0/task.json

```diff
@@ -19,7 +19,7 @@
     "version": {
         "Major": 0,
         "Minor": 2,
-        "Patch": 53
+        "Patch": 54
```

Comment:
Take into consideration the visibility rules for other fields. For example, 
1. SpecifySlot task input should not be visible as slot will be a mandatory field for this action
2. ResourceGroupName and Slot should be mandatory and visible fields for this action

Reply:
Why is "delete" made specifically for slot?
User can also delete "Production" slot, right? I agree this is a not the primary case in CI, but this would avoid visibility rule conflict.